ovn-tester/ovn_workload.py

```diff
@@ -142,17 +144,31 @@ def configure_localnet(self, physical_net):
 
     def configure_external_host(self):
         log.info(f'Adding external host on {self.container}')
-        gw_ip = netaddr.IPAddress(self.ext_net.last - 1)
-        host_ip = netaddr.IPAddress(self.ext_net.last - 2)
+        gw_ip = self.ext_net.get_n4_ip_reverse()
+        host_ip = self.ext_net.get_n4_ip_reverse(2)
+        host_prefix = self.ext_net.get_n4_prefix()
+        gw_ip6 = self.ext_net.get_n6_ip_reverse()
+        host_ip6 = self.ext_net.get_n6_ip_reverse(2)
+        host_prefix6 = self.ext_net.get_n6_prefix()
 
         self.run(cmd='ip link add veth0 type veth peer name veth1')
         self.run(cmd='ip netns add ext-ns')
         self.run(cmd='ip link set netns ext-ns dev veth0')
         self.run(cmd='ip netns exec ext-ns ip link set dev veth0 up')
-        self.run(cmd=f'ip netns exec ext-ns '
-                 f'ip addr add {host_ip}/{self.ext_net.prefixlen} '
-                 f'dev veth0')
-        self.run(cmd=f'ip netns exec ext-ns ip route add default via {gw_ip}')
+        if host_ip and host_prefix:
+            self.run(cmd=f'ip netns exec ext-ns '
```

Comment:
Same as above, IMO it would be better if we could extract it into single function.

Reply:
@almusil doing so we minimize the # of ssh calls

Reply:
@almusil ack, I will fix it